scripts/jenkins/ci.suse.de/openstack-mkcloud.xml

```diff
@@ -250,16 +250,13 @@ export want_mtu_size=8900
 export ZYPP_LOCK_TIMEOUT=120
 
 case $cloudsource in
-    GM4|GM4+up|GM5|GM5+up)
+    GM4|GM4+up|GM5|GM5+up|develcloud7|susecloud7)
         echo "Not setting want_ldap=1"
         ;;
     develcloud6)
         echo "Setting want_ldap=0 for upgrade jobs until hybrid backend is ported to Mitaka"
         [[ $mkcloudtarget =~ upgrade ]] &amp;&amp; export want_ldap=0
         ;;
-    M*|develcloud7|susecloud7)
-        echo "Not setting want_ldap=1 until hybrid backend is ported to Mitaka"
-        ;;
```

Comment:
I'm not really clear how this helps? This seems to be just moving code (and I actually like the fact that the temporary bit for Cloud 7 is split).

Also, `$cloudsource` is supposed to be `develcloud6` in the upgrade case. I'm just not sure why we don't enter the `develcloud6` case in the code, but instead the `*` one...